### DISCUSSION TITLE
Graph QL API Reference?

### DISCUSSION THREAD

**Discussion by lospec** [requarks/wiki#5671] [Help / Questions] [answered]
https://docs.requarks.io/dev/api

Doesn't seem to have an actual reference, just a few examples. Like how do I know which fields I can query on which thing?

**Comment by lospec** [comment, marked as answer]
seems the docs are only available on your own wiki at `/graphql`

**Reply by joshua-martius** [reply]
Does this work for you? I only get the message of "GET query missing". I found a lot of comments mentioning the graphql playground but I dont seem to be able to access it. My goal is to edit a page via the graphql endpoint

**Reply by NGPixel** [reply]
@joshua-martius You're most likely running node.js in production mode. Set `NODE_ENV` to `development` and you should be able to load the `/graphql` playground correctly.

**Reply by joshua-martius** [reply]
Yes that works! Oh my gosh, this is so satisfying! Loving Wiki.js even more right now as I can now create automatically update pages. Amazing!

Is the neccessity of switching to development mode documented anywhere? I couldnt find it.

**Reply by NGPixel** [reply]
No as the default is to not set that variable. This means you or some other process has set the variable to "production".

**Reply by joshua-martius** [reply]
Interesting. I am using the following docker image:
https://hub.docker.com/r/linuxserver/wikijs
It seems that they ship their container in production mode. I dont remember why I use that container instead of your official one but its good to know that.
